<launch>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find plan_env)/maps/astar_demo.rviz"/>
    <node pkg="map_server" type="map_server" name="map_server" args="$(find plan_env)/maps/map.yaml"/>
    <!-- <node pkg="a_star" type="a_star_demo" name="a_star_demo" output="screen"/> -->
    <!-- <node pkg="hybrid_a_star" type="hybrid_a_star_demo" name="hybrid_a_star_demo" output="screen">
        <param name="/steering_angle" value="15.0"/>
        <param name="/steering_angle_discrete_num" value="1"/>
        <param name="/wheel_base" value="2.0"/>
        <param name="/segment_length" value="1.6"/>
        <param name="/segment_length_discrete_num" value="8"/>
        <param name="/steering_penalty" value="1.5"/>
        <param name="/reversing_penalty" value="3.0"/>
        <param name="/steering_change_penalty" value="2.0"/>
        <param name="/shot_distance" value="5.0"/>
    </node> -->
    <node pkg="hybrid_a_star" type="hybrid_a_star_demo" name="hybrid_a_star_demo" output="screen"/>
</launch>